views/page/reporting/courseCompletions/choose-courses.html

```diff
@@ -0,0 +1,100 @@
+    {{ errorSummary(errors.fields, true) }}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-m">Reporting</span>
+    <h1 class="govuk-heading-l">{{ i18n.reporting.course_completions.title }}</h1>
+    <form method="post" action="/reporting/course-completions/choose-courses">
+        <div class="govuk-form-group">
+        {% set requiredLearningOptions = pageModel.requiredLearningList %}
+        {% if pageModel.requiredLearningList | length > 1 %}
+            {% set requiredLearningOptions = requiredLearningOptions.concat({
+                "divider": "or"
+            }) %}
+            {% set requiredLearningOptions = requiredLearningOptions.concat({
+                "value": "all",
+                "text": "All required learning",
+                "behaviour": "exclusive"
+            }) %}
+        {% endif %}
+        <input type='hidden' name='allRequiredLearning' value='{{ pageModel.allRequiredLearning }}'>
+      {% set requiredHtml %}
+        {{ govukCheckboxes({
+          name: "requiredLearning",
+          classes: "govuk-checkboxes--small",
+          fieldset: {
+            legend: {
+              text: "Required learning"
+            }
+          },
+          items: requiredLearningOptions
+        }) }}
+      {% endset -%}
+      {% set courseSearch %}
+        <div id="course-completions-autocomplete" class="autocomplete-wrapper--courses">
+          <label class="govuk-label" for="courseSearch">All courses</label>
+          <div id="subject-hint" class="govuk-hint">
+              <span id="course-completions-js-hint" class='hidden'>Start typing to filter available courses.</span> You can select up to {{ pageModel.maxCoursesSelection}} courses.
+          </div>
+          <select id="courseSearch" name="courseSearch" class="govuk-input" aria-labelledby="courseSearch" multiple>
+              {% for course in pageModel.courseSearchList %}
+                  <option {% if  (pageModel.courseSearch or []).indexOf(course.value) > -1 %}selected{% endif %} value={{ course.value }}>{{ course.text }}</option>
+              {% endfor %}
+          </select>
+        </div>
+      {% endset -%}
+      {{ govukRadios({
+        name: "learning",
+        fieldset: {
+          legend: {
+            text: "What courses would you like to report on?",
+            classes: "govuk-fieldset__legend--s"
+          }
+        },
+        items: [
+          {
+            value: "requiredLearning",
+            text: "Required learning for " + pageModel.userDepartment,
+            conditional: {
+                html: requiredHtml
+            },
+            checked: true
+          },
+          {
+              value: "courseSearch",
+              text: "Search for courses",
+              conditional: {
+                  html: courseSearch
+              }
+          }
+        ]
+      }) }}
+        </div>
+      {{ govukButton({
+        text: "Continue"
```

Comment:
Would it be a bit safer to include `preventDoubleClick: true` in the Continue button?

Also how about we label the button "Continue to completions overview" (or whatever the name of the graph page is)